src/Microsoft.IdentityModel.Tokens/AsymmetricSecurityKey.cs

```diff
@@ -27,8 +27,14 @@
 
 namespace Microsoft.IdentityModel.Tokens
 {
+    /// <summary>
+    /// base class for Asymmetric Security Key.
+    /// </summary>
     public abstract class AsymmetricSecurityKey : SecurityKey
     {
+        /// <summary>
+        /// true if there is a private key; otherwise, false.
```

Comment:
+1, since it's abstract it should read: "This must ...." rather than "This can ..."